src/main/target/ALIENFLIGHTF4/system_stm32f4xx.c

```diff
@@ -433,7 +433,7 @@ uint32_t hse_value = HSE_VALUE;
   */
 
 #if defined(STM32F40_41xxx)
-  uint32_t SystemCoreClock = 168000000;
+  uint32_t SystemCoreClock = 192000000;
```

Comment:
Isn't this overclocking the device? I thought they were only 168mhz, and 192kb ram?